tests/network/vmi_passt.go

```diff
@@ -43,6 +45,10 @@ var _ = SIGDescribe("[Serial] Passt", func() {
 	var err error
 	var virtClient kubecli.KubevirtClient
 
+	var _ = BeforeSuite(func() {
```

Comment:
`BeforeSuite` is usually outside of the container node (i.e `Describe`)

Reply:
Why do we need BeforeSuite and how does this work?

Reply:
Btw it must be outside
It runs just one time per suite
```
It looks like you are trying to add a BeforeSuite node within a container
BeforeSuite can only be called at the top level.
Learn more at:
http://onsi.github.io/ginkgo/#suite-setup-and-cleanup-beforesuite-and-aftersuite
```

Npte: that it actually might run only one time per all the tests, so need to make sure it doesn't skip everything when passt is disabled and then it will be wrong.

Reply:
>Note: that it actually might run only one time per all the tests, so need to make sure it doesn't skip everything (even unrelated to passt) when passt is disabled and then it will be wrong.

It does skip everything.

Reply:
[moved](https://github.com/kubevirt/kubevirt/compare/0476155a258824d691541b5fa753d037a684ea74..3c460abaa6baedab9be95c8ceccd26f1f7f152d0) it back to BeforeEach.